src/test/java/board/BoardTest.java

```diff
@@ -32,12 +33,7 @@ public void createNewBoard() {
         String boardId = json.get("id");
 
         // Remove previously created Trello board
-        given()
-                .spec(reqSpec)
-                .when()
-                .request(Method.DELETE, BOARDS + boardId)
-                .then()
-                .statusCode(200);
+        cleanUp(BOARDS + boardId);
```

Comment:
This is a great idea, I was about to tell you about that. I would rename method cleanUp to deleteResource as it is more  descriptive.

-- | --

Reply:
Done :)